kroxylicious/src/main/java/io/kroxylicious/proxy/KafkaProxy.java

```diff
@@ -141,12 +150,14 @@ else if (KQueue.isAvailable()) {
             workerGroup = new NioEventLoopGroup(availableCores);
             channelClass = NioServerSocketChannel.class;
         }
+
+        optionallyStartMetricsListener(bossGroup, workerGroup, channelClass);
```

Comment:
`maybe` is better than `optionally` imho. (It's also the convention that Apache Kafka uses, which is a minor additional benefit).

Reply:
switched to maybe